lib/coap/message.ex

```diff
@@ -204,7 +204,7 @@ defmodule CoAP.Message do
 
   def response_for(%__MODULE__{type: :con} = message) do
     %__MODULE__{
-      type: :con,
```

Comment:
This con never seemed right to me, but I'm pretty sure I copied it from `gen_coap`. Happy to merge this and find out! 😄

Reply:
Hmm, strange.  gen_coap responds with ack as far as I can tell